src/components/OptionList/components/Option/Option.scss

```diff
@@ -116,3 +117,53 @@ $control-vertical-adjustment: rem(2px);
   @include recolor-icon(color('ink', 'light'), color('white'));
   padding: 0 spacing(tight);
 }
+
+.globalTheming {
+  &.Option {
+    &:hover,
+    &:focus {
+      background: var(--p-surface-hovered);
+    }
+  }
+
+  .SingleSelectOption {
+    @include focus-ring;
+    // stylelint-disable-next-line selector-max-specificity,
+    &.focused:not(:active):not(.select) {
+      @include focus-ring($style: 'focused');
+    }
+  }
+
+  .focused {
+    box-shadow: none;
+    background: var(--p-surface-hovered);
+    // stylelint-disable-next-line selector-max-specificity
+    &:hover {
+      box-shadow: none;
+      background: var(--p-surface-hovered);
+    }
+  }
+
+  .disabled {
+    color: var(--p-text-disabled);
+    background: var(--p-action-secondary-disabled);
```

Comment:
@danrosenthal We don't seem to have a `--p-surface-disabled`. Should we be adding one?